packages/template-retail-react-app/app/translations/README.md

```diff
@@ -2,30 +2,30 @@
 
-Several **npm scripts** are available to you that make it easier to use the CLI tool. See `pwa/package.json` for more details.
+Several **npm scripts** are available to you that make it easier to use the CLI tool. See `package.json` for more details.
 
--   To **extract the default messages** in your app, run `npm run extract-default-translations` to have all of the default messages extracted into their own JSON file.
--   To **compile the translations** from all of the locales, run `npm run compile-translations`.
--   To run both an extract and compile, run `npm run build-translations`
+- To **extract the default messages** in your app, run `npm run extract-default-translations` to have all of the default messages extracted into their own JSON file.
+- To **compile the translations** from all of the locales, run `npm run compile-translations`.
+- To run both an extract and compile, run `npm run build-translations`
 
 ## Supported Locales and Default Locale
 
-The default locale, supported locales, and currency settings are defined in `pwa/package.json` in an object called `l10n`.
+The default locale, supported locales, and currency settings are defined in `package.json` in an object called `l10n`.
 
-The locale ids in `pwa/package.json` follow the format supported by OCAPI and Commerce API: `<language code>-<country code>` as defined in this InfoCenter topic: [OCAPI localization 21.8](https://documentation.b2c.commercecloud.salesforce.com/DOC1/topic/com.demandware.dochelp/OCAPI/current/usage/Localization.html).
+The locale ids in `package.json` follow the format supported by OCAPI and Commerce API: `<language code>-<country code>` as defined in this InfoCenter topic: [OCAPI localization 21.8](https://documentation.b2c.commercecloud.salesforce.com/DOC1/topic/com.demandware.dochelp/OCAPI/current/usage/Localization.html).
 
 The currency code in `l10n.supportedCurrencies` and `l10n.supportedLocales[n].preferredCurrency` follow the ISO 4217 standard.
 
-**Important**: The supported locale settings in `pwa/package.json` must match the locale settings for your B2C Commerce instance. For more information about configuring locales on a B2C Commerce instance, see this InfoCenter topic: [Configure Site Locales](https://documentation.b2c.commercecloud.salesforce.com/DOC2/topic/com.demandware.dochelp/content/b2c_commerce/topics/admin/b2c_configuring_site_locales.html).
+**Important**: The supported locale settings in `package.json` must match the locale settings for your B2C Commerce instance. For more information about configuring locales on a B2C Commerce instance, see this InfoCenter topic: [Configure Site Locales](https://documentation.b2c.commercecloud.salesforce.com/DOC2/topic/com.demandware.dochelp/content/b2c_commerce/topics/admin/b2c_configuring_site_locales.html).
 
-Here’s an example of locale configuration in `pwa/package.json`:
+Here’s an example of locale configuration in `package.json`:
```

Comment:
by default, locale configurations are no longer in package.json, they are moved to the config folder.

Reply:
this readme refers to package.json a lot and we probably need to update all of them